charts/linkerd2-cni/values.yaml

```diff
@@ -52,6 +48,14 @@ tolerations:
 # for more information
 #nodeAffinity:
 
+# -|- Image section
+image:
+  # -- Docker image for the CNI plugin
+  name: "cr.l5d.io/linkerd/cni-plugin"
+  # -- Tag for the CNI container Docker image
+  version: "v1.0.0"
+  imagePullPolicy:
```

Comment:
Given you introduced `imagePullPolicy` here, you should make use of it in the `cni-plugin.yaml` template :wink: